### PR TITLE
feat(web-components): create css partials for typography styles

### PR DESCRIPTION
**Pull request by chrisdholt**
## Previous Behavior
Only individual tokens or the text component were available for applying typography appearances.

## New Behavior
This PR creates new css partials which can be interpolated into components or used within stylesheets within Fluent UI and for use by those building their own custom elements using the FAST css directive.

## Related Issue(s)
n/a

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/1c8eea58748cf761f4b1f648c677220cdc485d34'>1c8eea58748cf761f4b1f648c677220cdc485d34</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json:1]: ## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31164&commitId=bdecd69ec0b426ca66634bc28a099c44c8507a46&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.With Long Text.default.chromium.png|1703| Changed|
|Button.With Long Text.pressed.chromium.png|1460| Changed|
|Button.With Long Text - RTL.hover.chromium.png|1607| Changed|
|Button.With Long Text - RTL.pressed.chromium.png|1445| Changed|
|Button.With Long Text.hover.chromium.png|1624| Changed|
|Button.With Long Text - RTL.default.chromium.png|1686| Changed|

</details>


  <details><summary><b>Slider</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps.normal.chromium.png|1| Changed|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Review by eljefe223 — APPROVED** [review]

File: change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json (hunk 1 above)
Comment (spmonahan) [reply change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json:1]: @chrisdholt Based on the VR tests it looks like you may have missed a rule.

File: change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "type": "prerelease",
+  "comment": "create CSS partials for named typography styles",
+  "packageName": "@fluentui/web-components",
+  "email": "=",
```
Comment (spmonahan) [inline change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json:5]: Should be an actual email :)

File: change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json (hunk 1 above)
Comment (chrisdholt) [reply change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json:1]: > @chrisdholt Based on the VR tests it looks like you may have missed a rule.

This needs to be rerun. The VR regression in Button got merged a little over an hour ago. With the removal of the internals, we needed to set text-align: center.

File: change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json (hunk 2 above)
Comment (chrisdholt) [inline change/@fluentui-web-components-6c28acc1-a99b-4101-a17a-54d88ec1a057.json:5]:
```suggestion
  "email": "13071055+chrisdholt@users.noreply.github.com",

**Pull request merged**